x-pack/plugins/security/server/types.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import type { IRouter, RequestHandlerContext } from 'src/core/server';
+import type { LicensingApiRequestHandlerContext } from '../../licensing/server';
+
+/**
+ * @internal
+ */
+export interface SecurityRequestHandlerContext extends RequestHandlerContext {
+  licensing: LicensingApiRequestHandlerContext;
+}
+
+/**
+ * @internal
+ */
+export type SecurityPluginRouter = IRouter<SecurityRequestHandlerContext>;
```

Comment:
Super minor nit: the naming convention of these types across all plugins is `XRequestHandlerContext` and `XRouter`, but this type is named `SecurityPluginRouter`. Perhaps `SecurityRouter` would be more consistent? Just stood out to me as I was looking 🙂

Reply:
ok, renamed